packages/ra-data-graphql/src/index.js

```diff
@@ -58,6 +58,12 @@ export default async options => {
         ...otherOptions
     } = merge({}, defaultOptions, options);
 
+    if (override && process.env.NODE_ENV === 'production') {
```

Comment:
I think we still use the `override` option in the `dataProvider` of the `examples/graphcool-demo`